be/src/common/config.h

```diff
@@ -798,6 +798,11 @@ CONF_Int32(s3_transfer_executor_pool_size, "2");
 
 CONF_Bool(enable_time_lut, "true");
 
+// number of s3 scanner thread pool size
+CONF_Int32(doris_s3_scanner_thread_pool_thread_num, "48");
```

Comment:
DEFAULT VALUE 48 is too big, it will cost too many cpu

Reply:
changed to 16